pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -287,7 +287,13 @@ func (os *OpenStack) ExpandVolume(volumeID string, newSize int) error {
 		return err
 	}
 
-	createOpts := volumeexpand.ExtendSizeOpts{
+	if volume.Size == newSize {
```

Comment:
@kayrus  I think we could move this check https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/controllerserver.go#L558 , so when equal we dont need to call NodeExpandVolume , so we could return false?
2. We can check both equal and greater than , then we dont need to expand.

Reply:
Good point, but I don't like that volume details are fetched twice per action:
* https://github.com/kubernetes/cloud-provider-openstack/blob/8cb96459c236d114e9c5f7bfb0ba8fdce232f059/pkg/csi/cinder/controllerserver.go#L550
* https://github.com/kubernetes/cloud-provider-openstack/blob/8cb96459c236d114e9c5f7bfb0ba8fdce232f059/pkg/csi/cinder/openstack/openstack_volumes.go#L285

Reply:
+1, I noticed too. would be good if you could fix it as well.